pkg/controller/servicebindingrequest/bindinginfo.go

```diff
@@ -0,0 +1,63 @@
+package servicebindingrequest
+
+import (
+	"fmt"
+	"strings"
+
+	omlv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
```

Comment:
Shoud we rename "omlv1alpha1" to "olmalphav1" ?

Reply:
You're saying that because use the major version of the package when naming it is a good practice?